frontend/purposeful-frontend/utils/fetch_wrapper.js

```diff
@@ -111,17 +127,12 @@ export default async function fetchWrapper(
       console.error(
         `Bad request. Status Code: ${status} Message: ${res.statusText}`
       );
-      return;
     }
-    // if (json) {
-    //   // want json
-    //   return await res.json();
-    // }
-    return res.json();
+    return res;
```

Comment:
To see whether this response object was successful or not, would we check its status?

Reply:
Usually, the response is only ok when the status is a 2xx, so by checking if it is ok, we are checking if it is successful